plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityEquipment.java

```diff
@@ -59,7 +61,7 @@ public static void register() {
         // @group inventory
         // @description
         // Returns a ListTag containing the entity's equipment.
-        // Output list is boots|leggings|chestplate|helmet
+        // Output list is boots|leggings|chestplate|helmet|body
```

Comment:
Just to make sure, have you tested this? as this uses `getArmorContents` from the Spigot API, which may not necessarily contain it